orquesta/specs/types.py

```diff
@@ -10,6 +10,10 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+BOOLEAN = {
+    "type": "boolean"
+}
```

Comment:
The other defined types add value to the base type.  I prefer to not redefine the base type here.